src/main/java/com/cloudbees/jenkins/support/slowrequest/SlowRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.cloudbees.jenkins.support.slowrequest;
+
+import com.cloudbees.jenkins.support.timer.FileListCap;
+import hudson.ExtensionList;
+import hudson.ExtensionPoint;
+
+import java.io.IOException;
+import java.util.List;
+
+public abstract class SlowRequest implements ExtensionPoint {
```

Comment:
I also doubt the method is ideal. Maybe it should be `AsyncMonitoringRequest` or whatever